src/GeneratorV2/Writing/Writer.cs

```diff
@@ -384,39 +197,32 @@ private static void WriteNativeMethod(NativeFunction function, bool postfixName,
 
         private static void WriteOverloadMethod(Overload overload1, IndentedTextWriter indentedTextWriter, bool postfixNativeCall)
         {
-            // FIXME: This was(?) used to cull "overloads" that didn't get any .
+            // This is used to cull methods that didn't get any overloads.
```

Comment:
Have we confirmed this is actually used? How many functions get filtered out here?

Reply:
I have confirmed this, idk how many methods get filtered out. But everything breaks if it is commented out, and i think i saw atleast 10 methods being filtered out with this. I can readd the FIXME if you still think its more appropriate, but i would like to start removing the FIXME's where possible.